artikel/2015-07-31-Hitzewelle-Sommerspaß-oder-Klimawandel.md

```diff
@@ -0,0 +1,34 @@
+#Hitzewelle: Sommerspaß oder Klimawandel?
+
+http://www.dw.com/de/hitzewelle-sommerspa%C3%9F-oder-klimawandel/a-18616694
+
+der Artikel "Hitzewelle: Sommerspaß oder Klimawandel?" spricht ein
+interessantes und wichtiges Thema an, weil wir diesen Sommer so viel 
+Hitzwelle in Deutschland gehabt haben.
+
+Der Artikel geht um Klimawandel und die Möglichkeit, ob es existiert. In 
+2015 gab es rekorde Temperaturen und häufiger Hitzwelle in Deutschland. 
```

Comment:
Here there's again an issue with `Hitzewelle`. The term is singular, but you want to pluralize it with `häufig`. If this really is your intent, then it would be `häufige Hitzewellen`. However, `Hitzewelle` from semantic perspective already refers to a "wave of heat" - a somewhat lengthy event with very hot temperature. Therefore my suggestion would be to keep it singular but choose a different adjective, sth. like `eine enorme Hitzewelle`.